Admin/AbstractAdmin.php

```diff
@@ -3063,7 +3072,7 @@ protected function configureRoutes(RouteCollection $collection)
       *
       * @return array
       */
-     protected function configureBatchActions($actions)
+     protected function configureBatchActions(&$actions)
```

Comment:
Not a huge fan of calls by reference BTW : http://stackoverflow.com/questions/2157799/when-to-pass-by-reference-in-php